contracts/src/v0.8/ccip/test/applications/PingPong/PingPong.plumbing.t.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.24;
+
+import {PingPongDemo} from "../../../applications/PingPongDemo.sol";
+import {PingPongDappSetup} from "./PingPongDappSetup.t.sol";
+
+contract PingPong_plumbing is PingPongDappSetup {
+  function test_Fuzz_CounterPartChainSelector_Success(
+    uint64 chainSelector
+  ) public {
+    s_pingPong.setCounterpartChainSelector(chainSelector);
+
+    assertEq(s_pingPong.getCounterpartChainSelector(), chainSelector);
+  }
+
+  function test_Fuzz_CounterPartAddress_Success(
+    address counterpartAddress
+  ) public {
+    s_pingPong.setCounterpartAddress(counterpartAddress);
+
+    assertEq(s_pingPong.getCounterpartAddress(), counterpartAddress);
```

Comment:
should be a separate file for each of these functions `getCounterPartAddress()`, `getCounterpartChainSelector`, and `isPaused`

Reply:
https://github.com/smartcontractkit/chainlink/pull/15165/commits/ca63592985c489d791bb56212fc2850410c50715